project_euler/problem_80/sol1.py

```diff
@@ -0,0 +1,73 @@
+"""
+https://projecteuler.net/problem=80
+
+It is well known that if the square root of a natural number is not an integer,
+then it is irrational. The decimal expansion of such square roots is infinite
+without any repeating pattern at all.
+
+The square root of two is 1.41421356237309504880...,
+and the digital sum of the first one hundred decimal digits is 475.
+
+For the first one hundred natural numbers, find the total of the digital
+sums of the first one hundred decimal digits for all the irrational square roots.
+"""
+
+
+from decimal import Decimal, getcontext
+
+
+def sqrt_with_accuracy(num: int, accuracy: int) -> str:
+    """Function to take the square root of a number with given accuracy.
+    Returns the result as a string.
+
+    >>> sqrt_with_accuracy(2, 100)
+    '1.414213562373095048801688724209698078569671875376948073176679737990732478462107038850387534327641572'
+    """
+    getcontext().prec = accuracy + 5
+    ans = str(Decimal(num).sqrt())
+    return ans[: accuracy + 1]
+
+
+def digital_sum(num: str) -> int:
+    """Function to calculate the digital sum of a number.
+    The digital sum is the sum of the digits of the representation of a number
+    Returns the digital sum as an int.
+
+    >>> digital_sum("1.414213562373095048801688724209698078569" \
```

Comment:
\ is not needed inside () or [] or {}.  \ line continuation is strongly discouraged in PEP8 and other places.